src/user/pipes/user.trim.pipe.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ArgumentMetadata,
+  BadRequestException,
+  Injectable,
+  PipeTransform,
+} from '@nestjs/common';
+
+const EXCEPT_FIELD = ['password'];
+
```

Comment:
may be we add `INCLUEDED_FIELD` to select only trim field to preventing for loss performance